taipy/gui/viselements.json

```diff
@@ -895,6 +895,18 @@
                         "type": "str|number",
                         "default_value": "None",
                         "doc": "The height, in CSS units, of the metric."
+                    },
+                    {
+                        "name": "format",
+                        "type": "str",
+                        "default_value": "None",
```

Comment:
no need to specify a `default_value` if it's None

Reply:
resolved